sip-ri/src/main/java/gov/nist/core/NameValueList.java

```diff
@@ -57,13 +57,15 @@ public class NameValueList implements Serializable, Cloneable, Map<String,NameVa
 
     private Map<String,NameValue> hmap;
 
+    protected boolean isQuotedString; //@MukhaValeriia
```

Comment:
I don't think this is being used anywhere is it ?

Reply:
Yes, my bad. Removed this method in TargetDialog.java in previous commit and didn't update here